projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -183,10 +190,12 @@ public static function create_scheduled_update( $timestamp, $interval, $plugins
 	 * @return \WP_Error|bool True on success, WP_Error on failure.
 	 */
 	public static function delete_scheduled_update( $timestamp, $plugins ) {
-		// Be sure to clear the cron cache before removing a cron entry.
+		$ret = wp_unschedule_event( $timestamp, self::PLUGIN_CRON_HOOK, $plugins, true );
```

Comment:
Could we find a more descriptive variable name?

Reply:
Renamed.